src/blockchain/log-processors/order-filled.ts

```diff
@@ -116,13 +137,13 @@ export function updateOrdersAndPositions(db: Knex, augur: Augur, trx: Knex.Trans
   }, (err: Error|null, onContractData: OrderFilledOnContractData): void => {
     if (err) return callback(err);
     const { amount, creatorPositionInMarket, fillerPositionInMarket } = onContractData!;
-    const amountRemainingInOrder = convertFixedPointToDecimal(new BigNumber(amount!, 10).toFixed(), numTicks);
+    const amountRemainingInOrder = convertFixedPointToDecimal(amount, numTicks);
     const updateParams = amountRemainingInOrder === "0" ? { amount: amountRemainingInOrder, isRemoved: 1 } : { amount: amountRemainingInOrder };
```

Comment:
Not relevant to this PR, but I'd like to change isRemoved to be only for re-orgs. It's currently overloaded and we might not be able to easily tell between a market that was re-org'd out and one that was closed.